core/src/main/scala/org/bitcoins/core/wallet/utxo/InternalAddressTag.scala

```diff
@@ -0,0 +1,77 @@
+package org.bitcoins.core.wallet.utxo
+
+/**
+  * An AddressTagType that is native to Bitcoin-S.
+  * InternalAddressTagTypes are still usable when using Bitcoin-S
+  * as a dependency
+  */
+sealed trait InternalAddressTagType extends AddressTagType
+
+/**
+  * An AddressTag that is native to Bitcoin-S.
+  * InternalAddressTags are still usable when using Bitcoin-S
+  * as a dependency
+  */
+sealed trait InternalAddressTag extends AddressTag
+
+/** An unknown address tag type, should mostly be used for parsing external address tags */
+case class UnknownAddressTagType(name: String) extends InternalAddressTagType
```

Comment:
just a thought, not sure if this is something we want, but we could add a `require` to this that makes sure `name` is not known internally? Same thing for `UnknownAddressTag` (where we would check both not just name)

Reply:
This would help us avoid multiple representations